$MirOS: ports/aqua/qt4-mac/patches/patch-configure,v 1.1 2008/07/02 18:49:43 bsiegert Exp $
--- configure.orig	2008-07-29 12:23:30.000000000 +0200
+++ configure	2008-08-25 14:59:20.000000000 +0200
@@ -3920,9 +3920,9 @@ if true; then ###[ '!' -f "$outpath/bin/
         fi
         if [ "$PLATFORM_MAC" = "yes" ]; then
             if [ "$PLATFORM" = "macx-icc" ]; then
-                echo "export MACOSX_DEPLOYMENT_TARGET = 10.4" >>"$mkfile"
+                echo "MACOSX_DEPLOYMENT_TARGET = 10.4" >>"$mkfile"
 	    else
-                echo "export MACOSX_DEPLOYMENT_TARGET = 10.3" >>"$mkfile"
+                echo "MACOSX_DEPLOYMENT_TARGET = 10.3" >>"$mkfile"
 	    fi
             echo "CARBON_LFLAGS =-framework ApplicationServices" >>"$mkfile"
             echo "CARBON_CFLAGS =-fconstant-cfstrings" >>"$mkfile"
